waspc/cli/src/Wasp/Cli/Command/CreateNewProject/StarterTemplates.hs

```diff
@@ -7,78 +7,190 @@ module Wasp.Cli.Command.CreateNewProject.StarterTemplates
+
+basicStarterTemplate :: StarterTemplate
+basicStarterTemplate =
   LocalStarterTemplate $
     DirBasedTemplateMetadata
-      { _name = "basic",
-        _path = "basic",
-        _description = "Simple starter template with a single page."
+      { _path = [reldir|basic|],
+        _name = "basic",
+        _description = "Simple starter template with a single page.",
+        _buildStartingInstructions = \projectDirName ->
+          unlines
+            [ styleText $ "To run your new app, do:",
+              styleCode $ "    cd " <> projectDirName,
+              styleCode $ "    wasp db start"
+            ]
       }
 
+{- HLINT ignore openSaasStarterTemplate "Redundant $" -}
+
+openSaasStarterTemplate :: StarterTemplate
+openSaasStarterTemplate =
+  simpleGhRepoTemplate
+    ("open-saas", [reldir|.|])
+    ( "saas",
+      "Everything a SaaS needs! Comes with Auth, ChatGPT API, Tailwind, Stripe payments and more."
+        <> " Check out https://opensaas.sh/ for more details."
+    )
+    ( \projectDirName ->
+        unlines
+          [ styleText $ "To run your new app, do:",
+            styleCode $ "    cd " <> projectDirName FP.</> "app",
+            styleCode $ "    wasp db start",
+            styleText $ "Then open new terminal window/tab in that same dir and do:",
+            styleCode $ "    wasp db migrate-dev",
+            styleCode $ "    cp .env.server.example .env.server",
+            styleCode $ "    wasp start",
+            styleText $ "",
+            styleText $ "Check the README for additional guidance and the link to docs!"
+          ]
+    )
+
+{- HLINT ignore todoTsStarterTemplate "Redundant $" -}
+
+todoTsStarterTemplate :: StarterTemplate
+todoTsStarterTemplate =
+  simpleGhRepoTemplate
+    ("starters", [reldir|todo-ts|])
+    ( "todo-ts",
+      "Simple but well-rounded Wasp app implemented with Typescript & full-stack type safety."
+    )
+    ( \projectDirName ->
+        unlines
+          [ styleText $ "To run your new app, do:",
+            styleCode $ "    cd " ++ projectDirName,
+            styleCode $ "    wasp db migrate-dev",
+            styleCode $ "    wasp start",
+            styleText $ "",
+            styleText $ "Check the README for additional guidance!"
+          ]
+    )
+
+{- Functions for styling instructions. Their names are on purpose of same length, for nicer code formatting. -}
+
+styleCode :: String -> String
+styleCode = Term.applyStyles [Term.Bold]
+
+styleText :: String -> String
+styleText = id
+
+{- -}
+
+embeddingsStarterTemplate :: StarterTemplate
+embeddingsStarterTemplate =
+  simpleGhRepoTemplate
+    ("starters", [reldir|embeddings|])
+    ( "embeddings",
+      "Comes with code for generating vector embeddings and performing vector similarity search."
+    )
+    ( \projectDirName ->
+        "To learn how to set up and run your new app, check instructions in "
+          <> projectDirName FP.</> "README.md"
+    )
```

Comment:
I think we can use the same instructions as for OpenSaas template by looking at the README.md

Reply:
Yup, I enriched them in the same way!

I didn't want to do it initially because I thought `embeddings` are less interesting + thought instructions are too complex to show them here, but actually it works, and I now had a "framework" to follow, so I did it.